src/actions/UserActions.js

```diff
@@ -3,7 +3,8 @@ import { SET_USER, LOGOUT } from '../constants/Constants';
 export function setUser(userData) {
   const user = {
     email: userData.attributes.email,
-    name: userData.attributes.name
+    name: userData.attributes.name,
+    admin: userData.admin
```

Comment:
Can't remember why we have `.attributes` for the other fields, does admin not need it?

Reply:
I remember seeing that the `attributes` field is just a follow-up of the format passed from cognito login - it doesn't seem to be necessary for any specific functionality as I can see